iot_verifier/src/last_beacon.rs

```diff
@@ -26,13 +40,13 @@ impl LastBeacon {
         .await?)
     }
 
-    pub async fn get<'c, E>(executor: E, id: &[u8]) -> anyhow::Result<Option<Self>>
+    pub async fn get<'c, E>(executor: E, id: &PublicKeyBinary) -> anyhow::Result<Option<Self>>
     where
         E: sqlx::Executor<'c, Database = sqlx::Postgres>,
     {
         Ok(
             sqlx::query_as::<_, LastBeacon>(r#" select * from last_beacon where id = $1;"#)
-                .bind(id)
+                .bind(id.as_ref())
```

Comment:
id should probably be TEXT instead of an array, then you wouldn't have to do `.as_ref()` everywhere

Reply:
This is an existing db table where the id field in a `bytea`, don't think it is in the scope of the PR to change the column type.